C-Plus-Plus/cp/Three_Sum.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+Description :
+    
+	Three Sum Problem - In this problem, user will provide an array and we
+	                    have find wether there is any "three" numbers are 
+						present in the array who's sum is equal to the target 
+						provided.If it is present then it will show true else 
+						it will show false.
+
```

Comment:
Aligned it properly.  Rewrite it as,
```
Three Sum Problem - In this problem, user will provide an array and we
                    have find wether there is any "three" numbers are 
                    present in the array who's sum is equal to the target 
		    provided. If it is present then it will show true else 
                    it will show false.
```

Also fix typo mistake `wether` to `whether`

Reply:
okay !